README.md

```diff
@@ -1,10 +1,25 @@
 # Pividori Lab Onboarding Repository
 
 The repository is home to the onboarding information for the [Pividori Lab](https://pivlab.org/) at the University of Colorado School of Medicine.
-**See [`onboarding.md`](onboarding.md)** for the onboarding information.
 
-## Useful links
-* [Pividori Lab logos](https://github.com/pivlab/pivlab-website/tree/main/images/logo).
+# Checklist for new lab members
+* **Read the [`onboarding.md`](onboarding.md) document**.
+It has important information about the lab's mission, expectations, lab meetings, indiviudal meetings, source code, reproducibility, etc.
+Ask if you have questions.
+* As a member of the lab, you should be able to **claim a GitHub Pro account** for your personal user.
+This can provide free access to **GitHub Copilot**.
+Check out the [GitHub Global Campus](https://education.github.com/).
+* **Request access to the [lab's GitHub organization](https://github.com/pivlab)**.
+For this, Milton will need your GitHub user.
+* **Request access to the [lab's Slack workspace](https://pivlab.slack.com/)**.
+We need your email address to send you an invitation to join.
+* **Add yourself to the [lab's team webpage](https://pivlab.org/team/)**.
+For this, you need to fork the [lab's webpage GitHub repo](https://github.com/pivlab/pivlab-website), make the necessary changes, and then submit a Pull Request (PR).
+Follow [the documentation on how to add team members](https://greene-lab.gitbook.io/lab-website-template-docs/basics/team-members) and/or take a look at [this PR](https://github.com/pivlab/pivlab-website/pull/4).
+* (optional) **Add your recent publications to the lab's website**.
+If you want visitors to know more about your research, you can add your recent publications to the lab's website.
+This usually involves modifying the `_data/sources.yaml` file.
+See [the documentation on citations](https://greene-lab.gitbook.io/lab-website-template-docs/basics/citations) for more information.
 
```

Comment:
perhaps mention how to request access to the Alpine cluster or add the links with the information?
add how to set up CU VPN, anydesk VPN and rustdesk tunnel?